### ISSUE TITLE
False positive in FormattingPlaceable

### ISSUE THREAD

**Issue by mathjazz** [translate/translate#3368]
In the following example, "% s" is detected as a placeable:
"xx% support a Web" -> "xx<mark>% s</mark>upport a Web"

The regex should be able to detect the whitespace between % and s and abort.

More specifically, the bug is probably in:
https://github.com/translate/translate/blob/master/translate/storage/placeables/general.py#L150

**Comment by dwaynebailey** [comment]
This class handles sprintf, a space here is perfectly valid according to sprintf.  Likely you don't want to use that type of placeable in a context that isn't using printf style C variables.

Correctly space is only valid if the conversion specifier is in (a, A, d, e, E, f, F, g, G, or i).  So that could be an improvement to prevent that being detected.

Patches gladly accepted.

**Comment by unho** [comment]
Perhaps addressed in PR https://github.com/translate/translate/pull/3375

**Comment by unho** [comment]
No, not addressed in https://github.com/translate/translate/pull/3375 since that is about checks, not placeables.
